cmd/state-offline-installer/main.go

```diff
@@ -38,6 +38,7 @@ func main() {
 
 	// Handle things like panics, exit codes and the closing of globals
 	defer func() {
+		panics.RecoverLocale = "offline_installer_recover_message"
```

Comment:
This feels like shenanigans. Please consider changing the function definition to take a locale key. A default value can be provided in the panics pkg with special keys being defined in calling pkgs.